### PR TITLE
Add prosecution case CRUD

### PR DESCRIPTION
**Pull request by archferns**
## What
Add ProsecutionCase CRUD via a simple admin interface.

## Checklist

Before you ask people to review this PR:

- [x] Tests and linters should be passing
- [x] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [x] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [x] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/views/admin/prosecution_cases/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  <%= render 'prosecution_case_identifier', form: form %>
+  <hr>
+
+  <h3>Defendants:</h3>
+
+    <%= form.fields_for :defendants do |defendant| %>
+      <%= render 'defendant', defendant: defendant %>
+    <% end %>
+
+
```
Comment (mpw5) [inline app/views/admin/prosecution_cases/_form.html.erb:63]: Unnecessary whitespace on lines 62/63

File: app/views/admin/prosecution_cases/_prosecution_case_identifier.html.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  <div class="field">
+    <%= prosecution_case_identifier.label :prosecutionAuthorityId, 'Prosecution Authority Id' %>
+    <%= prosecution_case_identifier.text_field :prosecutionAuthorityId %>
+  </div>
+
+  <div class="field">
+    <%= prosecution_case_identifier.label :prosecutionAuthorityCode, 'Prosecution Authority Code' %>
+    <%= prosecution_case_identifier.text_field :prosecutionAuthorityCode %>
+  </div>
+<% end %>
```
Comment (mpw5) [inline app/views/admin/prosecution_cases/_prosecution_case_identifier.html.erb:24]: GitHub is flagging that there's no newline at the end of this file

File: app/views/admin/prosecution_cases/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<p id="notice"><%= notice %></p>
+
+<% @prosecution_case.attributes.each do |key, value| %>
+  <p>
+    <strong><%= key.humanize %></strong>
+    <%= value %>
+  </p>
+<% end %>
+
```
Comment (mpw5) [inline app/views/admin/prosecution_cases/show.html.erb:9]: Unnecessary whitespace

File: app/views/admin/prosecution_cases/_form.html.erb (hunk 1 above)
Comment (archferns) [reply app/views/admin/prosecution_cases/_form.html.erb]: Thank you, updated these.
